I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 335, in script
    supochan "QUEM SÃO VOCÊS?! EU NÃO VOU CONSERTAR SEU PC"
Exception: Sayer 'supochan' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 335, in script
    supochan "QUEM SÃO VOCÊS?! EU NÃO VOU CONSERTAR SEU PC"
  File "C:\Users\pandi\Desktop\FACULDADE DUMP\renpy-8.1.0-sdk\renpy\ast.py", line 890, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\pandi\Desktop\FACULDADE DUMP\renpy-8.1.0-sdk\renpy\ast.py", line 783, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'supochan' is not defined.

Windows-10-10.0.19041 AMD64
Ren'Py 8.1.1.23060707
OINE_Project_Test 1.0
Mon Jun 19 13:48:34 2023
